onnxruntime/core/providers/openvino/ibackend.h

```diff
@@ -6,21 +6,24 @@
 #include <memory>
 #define ORT_API_MANUAL_INIT
 #include "core/session/onnxruntime_cxx_api.h"
+#include "onnx_ctx_model_helper.h"
 
 namespace onnxruntime {
 namespace openvino_ep {
 
 class IBackend {
  public:
   virtual void Infer(OrtKernelContext* context) = 0;
+  virtual ov::CompiledModel& GetOVCompiledModel() = 0;
 };
 
 class BackendFactory {
  public:
   static std::shared_ptr<IBackend>
   MakeBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
               GlobalContext& global_context,
-              const SubGraphContext& subgraph_context);
+              const SubGraphContext& subgraph_context,
+              EPCtxHandler& ctx_handle);
```

Comment:
Should the handle be const here ?

Reply:
During the MakeBackend call, we change the state of the EPCtxHandler object (fill in the blob string) so it cannot be const here.